nova/core/roles/connection/defaults/main.yml

```diff
@@ -9,3 +9,7 @@ primary_network: "{{ interfaces | selectattr('connection', 'equalto', true) | fi
 connection_mode: "{{ primary_network.addresses | selectattr('connection', 'equalto', true) | map(attribute='mode') | first }}"
 connection_mode_dhcp: "{{ true if connection_mode | regex_search('.*dhcp.*') else false }}"
 connection_address: "{{ 'DHCP' if connection_mode_dhcp else primary_network.addresses | selectattr('connection', 'equalto', true) | map(attribute='address') | first | ansible.utils.ipaddr('address') }}"
+
+# By default MacOS does not have Python installed so it cannot gather facts
+macos: false
```

Comment:
Is it in use?

Reply:
Yes, @AllRWeak I think you are only using `macos_has_python`

Reply:
Yes since MacOS is such a special system I've redesigned it to work similar to Network OS by setting `macos: true` in the OS group vars. So the deploy would not fail it needs to be defined here also